src/api.js

```diff
@@ -486,6 +487,18 @@ let ensureTaskGroup = async (ctx, taskId, taskDef, res) => {
     }
   });
 
+  // Now we walso add the task to the group size counters as well
+  await ctx.TaskGroupSize.create({
+    taskGroupId,
+    taskId,
+    expires: new Date(taskDef.expires),
+  }).catch(err => {
+    // If the entity already exists, then we're happy no need to crash
+    if (!err || err.code !== 'EntityAlreadyExists') {
+      throw err;
+    }
```

Comment:
We need some check of taskDef.expires here... I suspect it has to exactly the same... as that is the check we do for  `task.Create`... see `_.isEqual(taskDef, def)`